products/accredited-representative-facing/engineering/ADRs/pilot-2122-backend.md

```diff
@@ -0,0 +1,25 @@
+# Benefits Accredited Representative Facing (ARF) Tools ADR for Simplied Pilot Iteration of 21-22 Backend in `vets-api`
+
+## Status
+
+`Proposed`
+
+## Context
+
+The ART team has been tasked with beginning a 21-22 pilot by February. With a team of mostly new developers and the holidays looming, we're best served cutting technical complexity in the pilot wherever we can. We believe that the POA request backend technical design can be simplified for the sake of the pilot, and then a more complex solution that makes more BGS/Lighthouse calls can be potentially implemented after we launch the pilot.
+
+## Decision
+
+For the purposes of efficient pilot implementation, we'll first get an end-to-end 21-22 solution working where POA request information is stored in the VA.gov database, rather than making Lighthouse calls to persist that information elsewhere.
+
+- When a Veteran/VA.gov user creates a new POA request, the request will be saved in the VA.gov database, but no Lighthouse API calls will be made.
+- When an ARP user views POA requests, the VA.gov database will be queried for open requests, but no Lighthouse API calls will be made.
+- When an ARP user accepts or declines a POA request, a Lighthouse API call will be made to change the POA, using v2 of the Power of Attorney API. The VA.gov database will also be queried and modified to indicate that the POA request is closed.
```

Comment:
https://staging-developer.va.gov/explore/api/benefits-claims/docs?version=current

Reply:
1. Does this API need to be modified to return the proc ID from BGS?
2. At what point(s) in development does the team anticipate being blocked by Lighthouse or BGS?

Reply:
Responses from Alex (doesn't have GH access to reply himself yet):
> Does this API need to be modified to return the proc ID from BGS?

I don't believe proc id is relevant anymore to ART with this design, @jvcAdHoc can confirm.

> At what point(s) in development does the team anticipate being blocked by Lighthouse or BGS?

Right now - we currently have tickets assigned to engineers to implement the accept/reject flow, and a target of being feature complete by end of the year, which would give us 2 months to focus on setting the stage for the pilot and getting through hurdles. We also likely need to pull a bunch of our capacity to work on the claims flow by the beginning of next year if we're going to have a claim form in production by June. LH's roadmap currently envisions delivering the APIs by the end of the year, so the beginning of next year would be a good time to reevaluate.

Reply:
> Does this API need to be modified to return the proc ID from BGS?

Nope. We shouldn't need a proc id for a pure vets-api implementation.

Reply:
- If you're referring to LH's auto establishment endpoint, then there isn't a proc id to return. The auto establishment endpoints do not create poa request objects, they skip right to updating the necessary flags for an entity to have the access/permissions they need to act as power of attorney.
- This ADR proposes skipping the poa request creation endpoint which is where the proc id would come into play. TIL LH [already updated](https://github.com/department-of-veterans-affairs/vets-api/pull/19210) the creation endpoint to return proc id so we should be covered either way.